kenya_compliance/kenya_compliance/overrides/server/stock_ledger_entry.py

```diff
@@ -35,7 +35,7 @@ def on_update(doc: Document, method: str | None = None) -> None:
         "regTyCd": "M",
         "custTin": None,
         "custNm": None,
-        "custBhfId": get_warehouse_branch_id(doc.warehouse) or None,
+        "custBhfId": get_warehouse_branch_id(doc.get('warehouse')) or None,
```

Comment:
This is the same difference. Use the get method if you anticipate the key being accessed can be None, and you can provide a default value. In this case, the warehouse property will always be bound, so no reason for the get method.